cmd/dra-example-kubeletplugin/cdi.go

```diff
@@ -89,18 +91,37 @@ func (cdi *CDIHandler) CreateClaimSpecFile(claimUID string, devices PreparedDevi
 		Devices: []cdispec.Device{},
 	}
 
-	gpuIdx := 0
-	for _, device := range devices {
+	for i, device := range devices {
```

Comment:
From these changes it seems as if the `GPU_` prefix should catch all relevant output.

Reply:
I wanted to avoid printing the sharing information for the first 2 examples.

Reply:
But one doesn't seem to set the envvars for those example, but maybe I'm missing something from the code.

Reply:
TimeSlicing is always on as the default (as mimiced from NVIDIA GPUs)